### PR TITLE
Feature/restrict resend transaction option to qubic transfers

### LINKED ISSUE #193: Resend action should be available for all the transactions that are not pending

### PR DESCRIPTION
**Pull request by ahmed-tarek-salem**
Fixes #193

### CODE REVIEW

File: lib/components/transaction_item.dart [hunk 1]
```diff
@@ -149,13 +106,17 @@ class TransactionItem extends StatelessWidget {
...
               ),
               // sally: commenting this temporary until we improve the resend to support all transfers
-              //if (appStore.currentQubicIDs
-              //      .any((e) => e.publicId == item.sourceId) &&
-              //item.getStatus() != ComputedTransactionStatus.pending)
-              //PopupMenuItem<CardItem>(
-              //value: CardItem.resend,
-              //child: Text(l10n.transactionItemButtonResend),
-              //),
+              if (appStore.currentQubicIDs
```
Comment (sallymoc) [inline lib/components/transaction_item.dart:109]: @ahmed-tarek-salem 
1. my comment does not apply anymore, please delete it
2. Resend only should be available for the non watch only accounts

File: lib/smart_contracts/qx_info.dart [hunk 2]
```diff
@@ -10,4 +10,6 @@
   static const issueAssetInputType = 1;
   /// Input type for transferring an asset
   static const transferAssetInputType = 2;
+  static const qutilAddress = "EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVWRF";
```
Comment (sallymoc) [inline lib/smart_contracts/qx_info.dart:13]: @ahmed-tarek-salem 
Qx is one smart contract.
QUtil is another smart contract.

File: lib/components/transaction_item.dart (hunk 1 above)
Comment (ahmed-tarek-salem) [reply lib/components/transaction_item.dart:109]: @sallymoc 

The watch-only account doesn't have an option for sending transactions, so it's excluded by default.

Comment (sallymoc) [reply lib/components/transaction_item.dart:96]: That is not what I see in my test

Comment (sallymoc) [reply lib/components/transaction_item.dart:96]: image

Comment (ahmed-tarek-salem) [reply lib/components/transaction_item.dart]: Yeah got it, resolved.

File: lib/smart_contracts/qx_info.dart (hunk 2 above)
Comment (ahmed-tarek-salem) [reply lib/smart_contracts/qx_info.dart]: Resolved.

File: lib/smart_contracts/qutil_info.dart [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class QutilInfo {
+  static const address =
+      "EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVWRF";
```
Comment (sallymoc) [inline lib/smart_contracts/qutil_info.dart:3]: @ahmed-tarek-salem  Address is wrong. Should be 
EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVWRF

**Pull request merged**